### ISSUE TITLE
Make the validation sets completely separate/not same sitting as the training data

### ISSUE THREAD

**Issue by Sentdex** [Sentdex/BCI#5]
Pulling random samples from the entire set causes over-fitting far too easily. 
Pulling random 10 second sessions from the sets also causes over-fitting, just to a lesser degree. 

Going to try instead to make the validation set the latest `n` sessions, with the goal of having the validation set be a completely different day/setting as any of the training data.

**Comment by moznpan** [comment]
hey @Sentdex . I didn't have much luck escaping over-fitting either, but I figured I'd still share my results.

I took a slightly different approach and used the average across the channel dimension, resized the resulting image to a 13x13 instead, and flattened that image for input into a mlp. Here's my notebook as a [gist](https://gist.github.com/rcox771/8b8b0f14354782ff21f1381f3d15176f).

I also tried:
 - averaging the matrices out over 5, 10, and 30 minute intervals; and
 - fitting UMAPs to everything, but couldn't get a decent separation on either set, so I didn't include them in the gist.

Anywho, here's 5 samples after the transform for each:
![image](https://user-images.githubusercontent.com/13604961/69000677-b7315d80-0890-11ea-9fb7-8042483ed6e1.png)

and my confusion matrices:
![image](https://user-images.githubusercontent.com/13604961/69000745-b77e2880-0891-11ea-93ce-9ccd57ba1865.png)

![image](https://user-images.githubusercontent.com/13604961/69000746-bea53680-0891-11ea-838b-67c892310a0d.png)

Looking forward to helping test out the next dataset iteration.

-Russell

**Comment by RaghavendraGaleppa** [comment]
Hello I am able to hit 70% validation accuracy on the data inside validation_data folder.
You can check it out here: https://colab.research.google.com/drive/1v_vis7XpTQE0ANDoxK1Jr8_mzBFSbjF-
